blog/content/post/2024-11-04-qubes-os-summit-post.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 'Qubes Os Summit 2024'
+abstract: 'The sixth edition of the Qubes OS Summit, organized by Invisible
+Things Lab and 3mdeb, took place at the end of September'
+cover: /covers/image-file.png
+author: marta.witkowska, piotr.król
+layout: post
+published: false
+date: 2024-11-04
+archives: "2024"
+
+tags:
+  - QubesOS
+  - conference
+categories:
+  - Miscellaneous
+  - OS Dev
+
+---
+
+## Qubes OS Summit 2024
+
+The sixth edition of the Qubes OS Summit, organized by Invisible Things Lab and
+3mdeb, took place at the end of September. This year's event, held from
+September 20th to 22nd, was hosted in Berlin. It was not only an in-person
+gathering but also streamed live. The previous year’s conference was also hosted
+in Berlin and included a live online streaming for remote participants from
+[the first](https://www.youtube.com/watch?v=_UxndcxIngw&t=2s) and
+[the second](https://www.youtube.com/watch?v=xo2BVTn7ohs&t=3s) day of the
+summit.
+
+The summit was highly productive, facilitating the exchange of significant
+knowledge and expertise among participants. Contributions and insights shared
+during the event were greatly appreciated, as they advanced discussions on
+security, privacy, and open-source technologies.
```

Comment:
There is way too much exaggeration here. Avoid words like: `significant` and `greatly` when unjustified. When there are too many such words, read get the feeling of [BLIP](https://docs.dasharo.com/dev-proc/documentation-guidelines/#ai-usage)